kore/src/Kore/Internal/TermLike/TermLike.hs

```diff
@@ -724,13 +724,13 @@ instance (Unparse variable, Ord variable) => Unparse (TermLike variable) where
                             " */"
                   where
                     representation =
-                        addFunctionalRepresentation $
+                        addTotalRepresentation $
                             addFunctionRepresentation $
                                 addDefinedRepresentation $
                                     addSimplifiedRepresentation $
                                         addConstructorLikeRepresentation []
-                addFunctionalRepresentation
-                    | Attribute.isFunctional $ termFunctional attrs = ("Fl" :)
+                addTotalRepresentation
+                    | Attribute.isTotal $ termTotal attrs = ("Fl" :)
```

Comment:
```suggestion
                    | Attribute.isTotal $ termTotal attrs = ("T" :)
```

You might need to regenerate some integration test output to add this change.

Reply:
I'm going to do this in separate PR